go/vt/vtgate/planbuilder/jointree.go

```diff
@@ -95,6 +97,10 @@ type (
 
 		// columns needed to feed other plans`
 		columns []*sqlparser.ColName
+
+		// The following two fields are used when routing information_schema queries
+		SysTableTableSchema []evalengine.Expr
```

Comment:
hmm... we are using `sqlparser.Expr` for predicates here. feels icky to mix both parser stuff and evalengine stuff here. would it make sense to keep this as AST elements and turn them into evalengine thingies in the `jointree_transformers`?